### PR TITLE
hotfix/JM-7181 Unable to reassign response multi

### PR DESCRIPTION
**Pull request by akikrahman1**
### JIRA link (if applicable) ###

https://centralgovernmentcgi.atlassian.net/browse/JM-7181

### Change description ###

Updating code to allow paper and digital responses to be processed for reassignment between staff

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/juror/api/bureau/service/UserServiceImpl.java [hunk 1]
```diff
@@ -317,7 +330,11 @@ public void assignUrgentResponse(final DigitalResponse urgentJurorResponse) thro
             updateResponse.setStaffAssignmentDate(now.toLocalDate());
 
             log.trace("Assigning juror response {}", urgentJurorResponse);
-            jurorResponseRepository.save(updateResponse);
+            if (updateResponse.getReplyType().getType().equals(ReplyMethod.DIGITAL.getDescription())) {
+                jurorResponseRepository.save((DigitalResponse) updateResponse);
+            } else {
+                jurorPaperResponseRepository.save((PaperResponse) updateResponse);
+            }
```
Comment (Ben-Edwards-cgi) [inline src/main/java/uk/gov/hmcts/juror/api/bureau/service/UserServiceImpl.java:337]: Could this be pulled out into its own method?

(As it the same logic is used multiple times)

File: src/main/java/uk/gov/hmcts/juror/api/bureau/service/UserServiceImpl.java [hunk 2]
```diff
@@ -417,21 +417,9 @@ public AssignmentsListDto getStaffAssignments(AssignmentsMultiRequestDto respons
...
-                :
-                    "");
-            jurorDisplayName.append((jurorLastName != null)
-                ?
-                " " + jurorLastName
-                :
-                    "");
+            jurorDisplayName.append((jurorTitle != null) ? jurorTitle : "");
+            jurorDisplayName.append((jurorFirstName != null) ? " " + jurorFirstName : "");
+            jurorDisplayName.append((jurorLastName != null) ? " " + jurorLastName : "");
```
Comment (Ben-Edwards-cgi) [inline src/main/java/uk/gov/hmcts/juror/api/bureau/service/UserServiceImpl.java:422]: Enclosing braces not needed i.e 
((jurorTitle != null) ? jurorTitle : "")
should be
(jurorTitle != null ? jurorTitle : "")

etc

Comment (Ben-Edwards-cgi) [reply src/main/java/uk/gov/hmcts/juror/api/bureau/service/UserServiceImpl.java:422]: Also need to look when we prepend the space.

As if title is null we do not need the space before juror First Name, etc

**Review by Ben-Edwards-cgi — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/juror/api/bureau/service/UserServiceImpl.java (hunk 2 above)
Comment (akikrahman1) [reply src/main/java/uk/gov/hmcts/juror/api/bureau/service/UserServiceImpl.java]: removed brackets

**Pull request merged**